types/react-modal/index.d.ts

```diff
@@ -1,60 +1,43 @@
-// Type definitions for react-modal 1.6
+// Type definitions for react-modal 2.2
 // Project: https://github.com/reactjs/react-modal
-// Definitions by: Rajab Shakirov <https://github.com/radziksh>, Drew Noakes <https://github.com/drewnoakes>, Thomas B Homburg <https://github.com/homburg>
+// Definitions by: Guo Yunhe <https://github.com/guoyunhe>, Rajab Shakirov <https://github.com/radziksh>, Drew Noakes <https://github.com/drewnoakes>, Thomas B Homburg <https://github.com/homburg>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import * as React from "react";
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 
-export as namespace ReactModal;
-
-export = ReactModal;
-
-declare namespace ReactModal {
-    interface Styles {
-        content?: {
-            [key: string]: any;
-        };
-        overlay?: {
-            [key: string]: any;
-        };
-    }
+interface ModalProps {
```

Comment:
With this change you lose all of the related documentation previously on `Props`. It seems like it would be worthwhile to keep the documentation.